tiled/catalog/utils.py

```diff
@@ -1,7 +1,10 @@
+import hashlib
 import re
 from pathlib import Path
 from urllib.parse import urlparse, urlunparse
 
+import jcs
```

Comment:
I've never used a canonicalizing library in python, but does [conoicaljson](https://pypi.org/project/canonicaljson/) maybe look like a safer bet? It has declared `2.0.0` vs `0.21.x`, and doesn't rely on a different project to publish to PyPi (which is a over a year behind in the case of `jcs`)

Reply:
Good find, I had missed that one!

OK, I'll switch. Either way, the surface area is a single function call (`compute_structure_json`) so this will be a very easy refactor if we have to move in the future.

Reply:
Done!